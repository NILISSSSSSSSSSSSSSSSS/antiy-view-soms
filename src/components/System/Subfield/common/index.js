export default {
  deWeight: (item)=>{ //去除重复 级别的模块
    let obj = {}
    item = item.reduce((cur, next) => {
      if(!obj[next.levelType])
        obj[next.levelType] = true && cur.push(next)
      return cur
    }, [])
    return item
  },
  findPathByLeafId (leafId, nodes, next) { //遍历查找ID的父级
    let _t = this
    if(next === undefined) {
      next = []
    }
    for(let i = 0; i < nodes.length; i++) {
      let now = next.concat()  //浅拷贝
      now.push( nodes[i])
      if(leafId === nodes[i].stringId) {
        return now
      }
      if(nodes[i].childrenNode) {
        let findResult = _t.findPathByLeafId(leafId, nodes[i].childrenNode, now)
        if(findResult) {
          return findResult
        }
      }
    }
  }
}