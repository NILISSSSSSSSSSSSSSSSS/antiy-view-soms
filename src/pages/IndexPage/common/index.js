import { Fragment } from 'react'

const ASSETS_STATUS = [{
  name: '资产登记',
  value: 1
}, {
  name: '模板实施',
  value: 3
}, {
  name: '结果验证',
  value: 4
}, {
  name: '入网实施',
  value: 5
}, {
  name: '安全检查',
  value: 7
}, {
  name: '安全整改',
  value: 8
}, {
  name: '退役发起',
  value: 6
}, {
  name: '退役实施',
  value: 10
}]

export function onRow (ob, record) {
  let assetId, configId, bugpatchId, bugpatchInfoId = ''
  if (!record.businessId.includes('&') && record.businessId) {
    assetId = record.businessId //资产跳转
  } else if (record.businessId.split('&').length >= 3) {
    let len = record.businessId.split('&').length
    configId = record.businessId.split('&')[len - 1] //配置跳转
  } else if (['漏洞退回', '补丁处置', '补丁退回'].includes(record.name)) {
    bugpatchInfoId = record.businessId.split('&')[1]
  } else {
    bugpatchId = record.businessId.split('&')[0]//漏洞补丁跳转
  }
  switch (record.name) {
    case '基准配置':
      ob.push('/basesetting/manage?businessId=' + configId)
      break
    case '基准加固':
      ob.push('/basesetting/list/validation?businessId=' + configId)
      break
    case '基准核查':
      ob.push('/basesetting/list/enforcement?businessId=' + configId)
      break
    case (ASSETS_STATUS.some(item => item.name === record.name) ? record.name : ''):
      let init = ASSETS_STATUS.filter(item => item.name === record.name)[0].value
      ob.push(`/asset/manage?assetStatusList=${init}&conditionShow=true&id=` + assetId)
      break
    case (record.name.indexOf('装机模板') > -1 ? record.name : ''):
      ob.push('/asset/installtemplate?id=' + assetId)
      break
    case (['漏洞修复', '漏洞处置'].includes(record.name) ? record.name : ''):
      ob.push('/bugpatch/bugmanage/dispose?id=' + bugpatchId)
      break
    case '漏洞退回':
      ob.push('/bugpatch/bugmanage/information?id=' + bugpatchInfoId)
      break
    case '补丁安装':
      ob.push('/bugpatch/patchmanage/install?stringid=' + bugpatchId)
      break
    case '补丁处置':
      ob.push('/bugpatch/patchmanage/information?stringid=' + bugpatchInfoId)
      break
    case '补丁退回':
      ob.push('/bugpatch/patchmanage/information?stringid=' + bugpatchInfoId)
      break
    default:
      break
  }
}
//筛选流程类型
export function filterType (iData) {
  iData.forEach(item => {
    let init = item.processDefinitionId.split(':')[0]
    if (['assetAdmittance', 'installTemplate', 'assetRetire'].includes(init)) {
      item.typeName = '资产管理'
    } else if (['baselineConfig'].includes(init)) {
      item.typeName = '配置管理'
    } else if (['vulRepair'].includes(init)) {
      item.typeName = '漏洞管理'
    } else if (['patchInstall'].includes(init)) {
      item.typeName = '补丁管理'
    }
  })
  return iData
}
//大于四个字符返回万为单位
export const CharRegroup = (v)=>{
  if(!v) return 0
  v = String(v)
  if(v.length > 4) v = (
    <Fragment>
      {v.slice(0, v.length - 4)}
      <span style={{ display: 'inline-block', fontSize: '16px' }}>万</span>
    </Fragment>
  )
  return v
}

//图表配置
export const doughnutChart = {
  Dchart1: {
    order: 1,
    id: 'circle-chart-1',
    color: ['#3B6CFF', '#5BCDFF', '#AD7AFF', '#05CC7F', '#FFC438'],
    service: 'getHardware',
    title: '资产类型分布'
  },
  Dchart2: {
    order: 2,
    id: 'circle-chart-2',
    color: ['#4083FF', '#46ECFF', '#A569FF'],
    service: 'getAssetLevel',
    title: '资产重要程度分布'
  },
  Dchart3: {
    order: 3,
    id: 'circle-chart-3',
    color: ['#C22A4E', '#FF8E64', '#FFDA38', '#4083FF'],
    service: 'getWarningData',
    title: '告警等级分布'
  },
  Dchart4: {
    order: 4,
    id: 'circle-chart-4',
    color: ['#4083FF', '#46ECFF', '#A569FF', '#22B37A', '#FFDA38', '#FF8E64', '#4545FF'],
    service: 'getWarningDataChart',
    title: '告警类型分布'
  }
}