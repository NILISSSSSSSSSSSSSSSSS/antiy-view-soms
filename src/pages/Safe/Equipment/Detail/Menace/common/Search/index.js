import { Component } from 'react'
import CommonSearch from '@/components/common/Search'

const type = {
  apt: 'apt', // 高级威胁
  virus: 'virus', // 病毒威胁
  tanhai: 'tanhai'  //探海威胁
}
export default class Search extends Component {
  constructor (props) {
    super(props)
    this.state = {}
    this.defaultFields = [
      { label: '检索条件', key: 'retrieve', type: 'input', placeholder: 'IP/端口', data: [], maxLength: 200 },
      { label: '时间范围', key: 'time', type: 'dateRange', placeholder: ['开始日期', '结束日期'], data: [ ] }
    ]
  }
  // 重置查询
  handleReset = () => {
    const { reset, search } = this.props
    if(reset){
      reset()
    }else {
      search({})
    }
  }
  // 提交查询
  handleSubmit = (values = {}) => {
    let params = { ...values }
    if(values.time && values.time.length > 0) {
      const [ beginTime, endTime ] = values.time
      params.beginTime = beginTime ? beginTime.valueOf() : ''
      params.endTime = endTime ? endTime.valueOf() : ''
      delete params.time
    }
    params.multipleQuery = values.retrieve
    const { search } = this.props
    search && search(params)
  }
  /**
   * 给查询条件设置查询选项值
   * @param fields
   * @param list
   * @return {any[]}
   */
  setFilterData = (fields = [], list = []) => {
    return fields.map((e)=>{
      const currentData = list.find(it=>it.key === e.key) || {}
      if(e.key === currentData.key){
        return { ...e, ...currentData, data: currentData.data }
      }
      return e
    })
  }
  render () {
    const { placeholder, from } = this.props
    const { retrieve } = placeholder || {}
    let label = ''
    // 高级威胁
    if (from === type.apt){
      label = '追溯时间'
    }else if (from === type.virus){
      label = '中毒时间'
    }else if (from === type.tanhai){
      label = '最后活跃时间'
    }
    const fields = this.setFilterData(this.defaultFields, [{ key: 'retrieve', placeholder: retrieve }, { key: 'time', label }])
    return (
      <div className="search-bar">
        <CommonSearch wrappedComponentRef={search => { search && (this.searchForm = search.props.form) }} defaultFields={fields} onSubmit={this.handleSubmit} onReset={this.handleReset}/>
      </div>
    )
  }
}
