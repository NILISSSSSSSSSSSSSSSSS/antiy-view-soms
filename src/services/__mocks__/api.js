// GET 根据ID获取资产详情

export function getAssetHardWareById (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: { asset: { stringId: '-------------' } }
  })
}

export function batchRelation (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: { }
  })
}

export function claimTaskBatch (params) {
  return Promise.resolve({
    head: { code: '200' }
  })
}

export function querySuddenList (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: {
      items: [
        {
          antiyVulnId: '12345'
        }
      ],
      totalRecords: 1
    }
  })
}
export function getList (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: {
      items: [{}],
      totalRecords: 1
    }
  })
}
export function scanUrl (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: {
      onProgress: 10,
      errorMessage: '',
      findAssetNums: 10
    }
  })
}
export function getBugUsers (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: []
  })
}
export function onBackSubmit (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: []
  })
}
export function getAdvise (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: {}
  })
}
export function getVulnCpe (params) {
  return Promise.resolve({
    head: { code: '200' },
    body: {
      items: [],
      totalRecords: 1
    }
  })
}
export default {
  scanUrl,
  getAdvise,
  onBackSubmit,
  getBugUsers,
  getList,
  getAssetHardWareById,
  batchRelation,
  claimTaskBatch,
  querySuddenList,
  getVulnCpe
}
