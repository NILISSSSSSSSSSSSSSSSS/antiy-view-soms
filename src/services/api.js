import asset from './assets'
import * as workOrderApi from './workOrder'
import * as systemApi from './system'
import * as safeApi from './safe'
import * as organizationApi from './organization'
import * as taskPlan from './taskPlan'
import * as taskPlanReport from './taskPlanReport'
import * as bug from './bug'
import log from  './log'
import * as knowledge from  './knowledge'
import * as inspection from  './inspection'
import alarm from './alarm'
import * as baseSetting from  './baseSetting'
import * as indexPage from  './indexPage'
import * as workBench from  './workBench'
import * as reportForm from  './reportForm'
import  personalCenter from  './personalCenter'
import patch from './patch'
import * as defend from  './defend'
import * as examine from  './examine'
export default {
  ...patch,
  ...workBench,
  ...indexPage,
  ...asset,
  ...workOrderApi,
  ...systemApi,
  ...taskPlan,
  ...taskPlanReport,
  ...organizationApi,
  ...bug,
  ...log,
  ...knowledge,
  ...alarm,
  ...baseSetting,
  ...safeApi,
  ...inspection,
  ...reportForm,
  ...personalCenter,
  ...defend,
  ...examine
}
