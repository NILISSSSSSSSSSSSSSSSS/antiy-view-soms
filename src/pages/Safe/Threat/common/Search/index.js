import { Component } from 'react'
import api from '@/services/api'
import CommonSearch from '@/components/common/Search'

const type = {
  apt: 'apt', // 高级威胁
  virus: 'virus', // 病毒威胁
  tanhai: 'tanhai'  //探海威胁
}

export default class Search extends Component {
  constructor (props) {
    super(props)
    this.state = {
      equList: [] // 所属的安全设备列表
    }
    this.filterFields = [
    ]
    this.defaultFields = [
      { label: '所属安全设备', allowClear: false, placeholder: '全部', key: 'safetyId', type: 'select', config: { name: 'name', value: 'id' } },
      { label: '检索条件', key: 'retrieve', type: 'input', placeholder: '全部', data: [], maxLength: 200 },
      { label: '时间范围', key: 'time', type: 'dateRange', placeholder: ['开始日期', '结束日期'], data: [ ] }
    ]
  }
  componentDidMount () {
    const { from } = this.props
    // 高级威胁
    if (from === type.apt){
      this.getAPTEqu()
    }else if (from === type.virus){
      this.getVirusEqu()
    }else if (from === type.tanhai){
      this.getTanhaiEqu()
    }
  }

  // 获取高级威胁所属的安全设备
  getAPTEqu = () => {
    api.getsafetythreataptEquip().then((res)=>{
      if(res){
        this.setState({ equList: res.body })
      }
    })
  }
  // 获取探海威胁所属的安全设备
  getTanhaiEqu = () => {
    api.getsafetythreattanhaiEquip().then((res)=>{
      if(res){
        this.setState({ equList: res.body })
      }
    })
  }

  // 获取病毒所属的安全设备
  getVirusEqu = () => {
    api.getsafetythreatvirusEquip().then((res)=>{
      if(res){
        this.setState({ equList: res.body })
      }
    })
  }
  // 重置查询
  handleReset = () => {
    const { reset } = this.props
    reset && reset()
  }

  handleSubmit = (values = {}) => {
    let params = { ...values }
    if(values.time && values.time.length > 0) {
      const [ beginTime, endTime ] = values.time
      params.beginTime = beginTime ? beginTime.valueOf() : ''
      params.endTime = endTime ? endTime.valueOf() : ''
      delete params.time
    }
    params.safetyId = values.safetyId
    params.multipleQuery = values.retrieve
    const { search } = this.props
    search && search(params)
  }
  /**
   * 给查询条件设置查询选项值
   * @param fields
   * @param list
   * @return {any[]}
   */
  setFilterData = (fields = [], list = []) => {
    return fields.map((e)=>{
      const currentData = list.find(it=>it.key === e.key) || {}
      if(e.key === currentData.key){
        return { ...e, ...currentData, data: currentData.data }
      }
      return e
    })
  }
  render () {
    const { placeholder, from } = this.props
    const { safetyId, retrieve } = placeholder || {}
    const { equList } = this.state
    let label = ''
    // 高级威胁
    if (from === type.apt){
      label = '追溯时间'
    }else if (from === type.virus){
      label = '中毒时间'
    }else if (from === type.tanhai){
      label = '最后活跃时间'
    }
    const fields = this.setFilterData(this.defaultFields, [{ key: 'safetyId', data: equList, placeholder: safetyId }, { key: 'time', label }, { key: 'retrieve', placeholder: retrieve }])
    return (
      <div className="search-bar">
        <CommonSearch wrappedComponentRef={search => { search && (this.searchForm = search.props.form) }} defaultFields={fields} onSubmit={this.handleSubmit} onReset={this.handleReset}/>
      </div>
    )
  }
}
