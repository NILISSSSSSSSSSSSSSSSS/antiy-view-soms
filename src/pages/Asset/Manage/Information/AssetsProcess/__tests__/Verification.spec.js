import React from 'react'
import Verification  from '../Verification'
import { shallow } from 'enzyme'
import PropTypes from 'prop-types'
import { TooltipFn } from '@u/common'
import api from '../../../../../../services/api'

jest.mock('../../../../../../services/api')

const props = {
  onRef: jest.fn(),
  getAssetList: jest.fn()
}

api.getCheckReportAsset = jest.fn()
api.getCheckReportAsset.mockReturnValue(
  Promise.resolve({
    head: { code: '200', result: 666 },
    body: {  }
  })
)

const store = {
  subscribe: () => ({
  }),
  dispatch: () =>({}),
  getState: () => ({
    asset: {
      previousBatch: [
        {
          assetId: 666,
          note: '',
          fileInfo: ''
        }
      ],
      usersByRoleCodeAndAreaIdList: [
        {
          stringId: 666,
          name: 666
        }
      ]
    },
    staticAsset: {
      assetModelColumns: [
        {
          title: '资产名称',
          dataIndex: 'name',
          key: 'name',
          render: text=>TooltipFn(text)
        },
        {
          title: 'IP',
          dataIndex: 'ips',
          key: 'ips',
          render: text=>TooltipFn(text)
        }
      ]
    }
  })
}
const options = {
  context: { store },
  childContextTypes: { store: PropTypes.object.isRequired }
}

describe('<Verification />', () => {
  let wrapper, instance
  const recordsArr = [{
    stringId: 666,
    stepNode: 666,
    note: 666,
    fileInfo: null,
    waitingTaskReponse: {
      taskId: 666
    }
  }]

  beforeEach(() => {
    wrapper = shallow(<Verification {...props}/>).dive(options).dive(options)
    instance = wrapper.instance()
  })

  const testInit = ()=>{
    instance.openModel(true, recordsArr)
  }

  beforeEach(() => {
    testInit()
  })

  it('render <Verification /> component', () => {
    expect(wrapper.find('.process-wrap')).toHaveLength(1)
    instance.state.analysisCarry = true
  })

  it('test  shouldComponentUpdate', () => {
    const nextState1 = {
      visible: true
    }
    const nextState2 = {
      visible: false
    }
    instance.shouldComponentUpdate(null, nextState1)
    instance.state.visible = true
    instance.shouldComponentUpdate(null, nextState2)
  })

  it('testFunc openModel',  ()=> {
    instance.openModel(false)
  })

  it('testFunc UploadChange',  ()=> {
    const info = {
      fileList: [
        {
          name: 666,
          uid: 666,
          status: 'done',
          percent: 100,
          response: {
            head: {
              code: '200'
            }
          }
        }
      ]
    }
    instance.UploadChange(info)
  })

  it('testFunc changeReject',  ()=> {
    const spyFunc = jest.spyOn(instance, 'changeReject')
    spyFunc()
    expect(spyFunc).toHaveBeenCalled()
  })

  it('testFunc uploadChangeBatch',  ()=> {
    instance.uploadChangeBatch(666, true, true, true )
  })

  it('testFunc isTableFn',  ()=> {
    instance.isTableFn(jest.fn(), jest.fn() )
    const Arr = [
      ...recordsArr,
      {
        stringId: 667,
        stepNode: 666,
        note: 666,
        fileInfo: null,
        waitingTaskReponse: {
          taskId: 666
        }
      }]
    instance.openModel(true, Arr)

    instance.isTableFn(jest.fn(), jest.fn() )

  })

  it('testFunc uploadChange',  ()=> {
    const info = {
      fileList: [
        {
          name: 666,
          uid: 666,
          status: 'uploading',
          percent: 100,
          response: {
            head: {
              code: '200'
            }
          }
        },
        {
          name: 666,
          uid: 666,
          status: 'done',
          percent: 100,
          response: {
            head: {
              code: '200'
            }
          }
        }
      ]
    }
    instance.uploadChange(info)
  })

  it('testFunc lookReportAlone',  ()=> {
    instance.lookReportAlone()
  })

  it('testFunc lookReportBatch',  ()=> {
    instance.lookReportBatch(666)
  })

  it('testFunc lookReportBatch',  ()=> {
    instance.lookReportBatch(recordsArr)
  })

  it('testFunc downTempAlone',  ()=> {
    instance.downTempAlone()
  })

  it('testFunc downTempBatch',  ()=> {
    instance.downTempBatch(...recordsArr)
  })

  it('testFunc delete',  ()=> {
    instance.delete(666)
  })

  it('testFunc isDeleteCB',  ()=> {
    instance.isDeleteCB()
  })

  it('testFunc downTemp',  ()=> {
    const params = {
      flag: 666,
      ids: 666,
      comIds: 666
    }
    instance.downTemp(params)
  })

  it('testFunc baselineFileAssetCB',  ()=> {
    const body = [
      {
        assetId: 666,
        result: true
      }
    ]
    instance.baselineFileAssetCB(recordsArr, body)
  })

  it('testFunc urlCB',  ()=> {
    instance.urlCB('/basesetting/list/enforcement?', 666)
  })

  it('testFunc queryProcessAssetCB',  ()=> {
    const res = {
      body: [
        {
          assetId: 666,
          onProgress: 1
        },
        {
          assetId: 6,
          onProgress: 1
        }
      ]
    }
    instance.queryProcessAssetCB(res)
  })

  it('testFunc UploadChange',  ()=> {
    const info = {
      fileList: [
        {
          name: 666,
          uid: 666,
          status: 'done',
          percent: 100,
          response: {
            head: {
              code: '200'
            }
          }
        }
      ]
    }
    instance.UploadChange(info)
  })

  it('testFunc previousBatch',  ()=> {
    instance.state.isTable = true
    const spyFunc = jest.spyOn(instance, 'previousBatch')
    spyFunc()
    expect(spyFunc).toHaveBeenCalled()
  })

  it('testFunc onSubmit',  ()=> {
    const spyFunc = jest.spyOn(instance, 'onSubmit')
    spyFunc()
    expect(spyFunc).toHaveBeenCalled()
  })

  it('testFunc onSubmitCB',  ()=> {
    const values = {
      file: {
        fileList: [
          {
            response: {
              body: [{
                fileUrl: 666,
                originFileName: 666
              }]
            }
          }
        ]
      },
      operator: 'all',
      agree: 1,
      note: 666
    }
    testInit()
    api.statusJump = jest.fn()
    api.statusJump.mockReturnValue(
      Promise.resolve({
        head: { code: '200', result: 666 },
        body: {  }
      })
    )
    const spyFunc = jest.spyOn(instance, 'onSubmitCB')
    spyFunc(values)
    expect(spyFunc).toHaveBeenCalled()
  })

  it('testFunc queryProcessAsset',  ()=> {
    api.queryProcessAsset = jest.fn()
    api.queryProcessAsset.mockReturnValue(
      Promise.resolve({
        head: { code: '200', result: 666 },
        body: {  }
      })
    )
    instance.queryProcessAsset()
  })

  it('testFunc getCheckReportAsset',  ()=> {
    // api.getCheckReportAsset = jest.fn()
    // api.getCheckReportAsset.mockReturnValue(
    //   Promise.resolve({
    //     head: { code: '200', result: 666 },
    //     body: {  }
    //   })
    // )
    instance.getCheckReportAsset()
  })

  it('testFunc baselineFileAsset',  ()=> {
    api.baselineFileAsset = jest.fn()
    api.baselineFileAsset.mockReturnValue(
      Promise.resolve({
        head: { code: '200', result: 666 },
        body: {  }
      })
    )
    instance.baselineFileAsset()
  })

  it('shoulder render a Verification',  ()=> {
    const app = wrapper.debug()
    expect(app).toMatchSnapshot()
  })

})
