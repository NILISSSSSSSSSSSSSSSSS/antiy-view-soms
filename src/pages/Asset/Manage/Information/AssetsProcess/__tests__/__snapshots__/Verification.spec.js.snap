// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Verification /> shoulder render a Verification 1`] = `
"<Connect(Form(commonModal)) title=\\"结果验证\\" type=\\"search\\" visible={true} width={680} oktext=\\"确定\\" onConfirm={[Function]} onClose={[Function: onClose]}>
  <div className=\\"AssetsProcess\\">
    <div className=\\"process-wrap\\">
      <div className=\\"modal-content\\">
        <section className=\\"operate-info\\">
          <h3>
            结果验证
          </h3>
          <p>
            请先对资产的漏洞情况和基准情况进行扫描。
          </p>
          <div className=\\"operate-content\\">
            <div className=\\"result-info\\">
              <img src=\\"test-file-stub\\" className=\\"item-icon\\" alt=\\"\\" />
              <span>
                漏洞检查结果：
              </span>
              <span className=\\"info\\">
                漏洞检查中,请等待
              </span>
            </div>
          </div>
          <br />
          <span className=\\"download\\">
            <img src=\\"test-file-stub\\" className=\\"download-icon\\" alt=\\"\\" />
            <a onClick={[Function]}>
              下载模板
            </a>
          </span>
          <span className=\\"operate\\">
            <Upload name=\\"file\\" accept=\\".xml\\" action=\\"/api/v1/config/check/uploadCheckResult\\" headers={{...}} multiple={false} showUploadList={false} onChange={[Function: onChange]} data={{...}} beforeUpload={[Function: beforeUpload]} disabled={false} type=\\"select\\" listType=\\"text\\" className=\\"\\" supportServerRender={true}>
              <span>
                上传结果
              </span>
            </Upload>
          </span>
        </section>
        <section>
          <h3>
            填写验证结果
          </h3>
          <Form layout=\\"horizontal\\" colon={true} hideRequiredMark={false} onSubmit={[Function: onSubmit]}>
            <FormItem labelCol={{...}} wrapperCol={{...}} label=\\"验证结果\\" hasFeedback={false}>
              <RadioGroup onChange={[Function: bound bound onCollectValidate]} disabled={false} buttonStyle=\\"outline\\" value={1} id=\\"agree\\" data-__meta={{...}} data-__field={{...}}>
                <Radio value={1} type=\\"radio\\">
                  通过
                </Radio>
                <Radio value={0} type=\\"radio\\">
                  拒绝
                </Radio>
              </RadioGroup>
            </FormItem>
            <FormItem labelCol={{...}} wrapperCol={{...}} label=\\"备注\\" hasFeedback={false}>
              <TextArea rows={6} maxLength={300} placeholder=\\"填写退役实施结果，或拒绝退役的原因\\" style={{...}} value={[undefined]} id=\\"note\\" onChange={[Function: bound bound onCollectValidate]} data-__meta={{...}} data-__field={{...}} />
            </FormItem>
            <FormItem labelCol={{...}} wrapperCol={{...}} label=\\"上传附件\\" hasFeedback={false}>
              <Upload name=\\"fileList\\" action=\\"/api/v1/asset/file/upload\\" beforeUpload={[Function: beforeUpload]} accept=\\".rar,.zip,.7z,.doc,.docx,.jpg,.pdf,.png,.txt,.xlsx,.xls\\" onChange={[Function: bound bound onCollect]} fileList=\\"\\" data={{...}} headers={{...}} type=\\"select\\" multiple={false} showUploadList={true} listType=\\"text\\" className=\\"\\" disabled={false} supportServerRender={true} value={[undefined]} id=\\"file\\" data-__meta={{...}} data-__field={{...}}>
                <div className=\\"work-order-add-upload\\">
                  <Icon type=\\"plus\\" />
                    上传附件  
                </div>
              </Upload>
              <div className=\\"format\\">
                <div>
                  <p>
                    支持上传大小：0-10M
                  </p>
                  <p>
                    支持扩展名：.rar .zip .7z .doc .docx .jpg .pdf .png .txt .xlsx .xls 
                  </p>
                </div>
              </div>
            </FormItem>
            <FormItem labelCol={{...}} wrapperCol={{...}} label=\\"入网实施执行人\\" hasFeedback={false}>
              <Select showSearch={true} allowClear={true} getPopupContainer={[Function: getPopupContainer]} placeholder=\\"请选择\\" optionFilterProp=\\"children\\" transitionName=\\"slide-up\\" choiceTransitionName=\\"zoom\\" value={[undefined]} id=\\"operator\\" onChange={[Function: bound bound onCollectValidate]} data-__meta={{...}} data-__field={{...}}>
                <Option value=\\"all\\">
                  全部
                </Option>
                <Option value={666}>
                  666
                </Option>
              </Select>
            </FormItem>
          </Form>
        </section>
      </div>
    </div>
  </div>
</Connect(Form(commonModal))>"
`;
